llvm/lib/Transforms/Utils/SSAUpdater.cpp

```diff
@@ -269,6 +270,13 @@ class SSAUpdaterTraits<SSAUpdater> {
                                SSAUpdater *Updater) {
     PHINode *PHI = PHINode::Create(Updater->ProtoType, NumPreds,
                                    Updater->ProtoName, &BB->front());
+
+    if (revng::RevngDebugInformationPreservationStyle) {
+      // Not the cleanest, but at the very least it ensures at least some kind
+      // of debug-information is present.
```

Comment:
Please provide context in the comment. Why is this "not clean"?

Reply:
```cpp
      // This is not the best, the proper solution would be to accept the debug
      // location from the callee, but this will do for now.
```